### DISCUSSION TITLE
Are we ready for more complex app / testing?

### DISCUSSION THREAD

**Discussion by bdeckerdev** [trobfab/fabrik#593] [General]
Hello all, reporting for testing duty on beta 3.  I did several hours last night and generated several lists with various element types and all went flawlessly.  Also tested some list plugins.  Wanted to say "Thank you" to this team.   I've also tested with the Rocket Hydrogen theme and have found no issues so far. 

I have a large and complex app in F3/J3 that I'd like to start porting.  I like to manually build it up from scratch. It allows me to clean up things as I go.  I wanted to ask if the developers are ready for more intense app testing.  I have no timeframe expectations but would be happy to dive in to see if I can break anything with my more complex app with lots of PHP hooks.

Best,
Bruce

**Comment by trob** [comment]
Go on ;)
Use the latest version, see https://github.com/joomlahenk/fabrik/wiki/Tester-Instructions
For upgrades see https://github.com/joomlahenk/fabrik/wiki/Upgrading-from-fabrik3.10-to-fabrik4
(also if you are "building from scratch" you will have to adapt your existing php code)

**Comment by skurvish** [comment]
@bdeckerdev sorry for the delay in responding to this, we do not get notifications of new topics in the discussion section (looking to see if we can get them). As @trob says, please proceed. Report any issues in the issues section. The current beta4 seems to be pretty stable.
